website/src/components/flex/Flex.js

```diff
@@ -0,0 +1,76 @@
+import cx from "classnames";
+import slugify from "slugify";
+import FlexHero from "./FlexHero";
+import FlexWysiwyg from "./FlexWysiwyg";
+import FlexCarousel from "./FlexCarousel";
+import FlexBlockquote from "./FlexBlockquote";
+
+import styles from "./Flex.module.scss";
+
+const Flex = ({ sections }) => {
+  const renderedSections = [];
+
+  for (let i = 0; i < sections.length; i++) {
+    let component, componentName;
+    let paddingBottom = true;
+    let paddingTop = true;
+    let darkBG = false;
+    const section = sections[i];
+    const nextSection = sections[i + 1];
+
+    // collapse padding if two sections in a row have the same color
+    if (
+      nextSection &&
+      nextSection.backgroundColor === section.backgroundColor
+    ) {
+      paddingBottom = false;
+    }
+
+    // for known dark bg's, we'll add white text
+    let backgroundDark = false;
+    const darkArr = ["primary", "black", "dark"];
+    if (darkArr.includes(section.backgroundColor)) {
+      backgroundDark = true;
+    }
+
+    if (typeof section.blockquote !== "undefined") {
```

Comment:
For this, instead of testing a sub field, can we test `fieldGroupName`? It looks consistent to me:

```
flexContent: [
    {
      backgroundColor: 'light',
      fieldGroupName: 'page_Acfflex_FlexContent_Hero',
      heroHeading: 'Heading',
      heroSubheading: 'subhead',
      sectionClasses: 'bg-blue bg-test',
      sectionSlug: 'hero Dawg'
    },
    {
      backgroundColor: 'light',
      fieldGroupName: 'page_Acfflex_FlexContent_WysiwygContent',
...
}
]
```

Reply:
Yes, i'll search those strings as well.